plugin/src/main/java/net/dzikoysk/funnyguilds/config/subcomponents/HeartConfiguration.java

```diff
@@ -0,0 +1,85 @@
+package net.dzikoysk.funnyguilds.config.subcomponents;
+
+import eu.okaeri.configs.OkaeriConfig;
+import eu.okaeri.configs.annotation.Comment;
+import eu.okaeri.configs.annotation.CustomKey;
+import eu.okaeri.configs.annotation.Exclude;
+import net.dzikoysk.funnyguilds.FunnyGuilds;
+import net.dzikoysk.funnyguilds.shared.bukkit.MaterialUtils;
+import org.apache.commons.lang3.tuple.Pair;
+import org.bukkit.Material;
+import org.bukkit.entity.EntityType;
+
+import java.io.File;
+
+public class HeartConfiguration extends OkaeriConfig {
```

Comment:
Jeśli już robić to jako oddzielna klasa, a nie klasa wewnętrzna PluginConfigurafion to może od razu zrobić to jako oddzielny plik (np. _heart.yml_).

Reply:
Zrobiłem to oddzielnie bo jest duży bałagan w głównej konfiguracji.